build/docker/README.md

```diff
@@ -180,7 +180,7 @@ Last updated: October 1, 2017
 | erlang    | OTP-18        | OTP-23        |       |
 | go        | 1.15.10       | 1.16.2        |       |
 | haxe      | 3.2.1         | 3.4.4         | THRIFT-4352: avoid 3.4.2 |
-| java      | 1.8.0\_191     | 11.0.3        |       |
+| java      | 1.8.0\_191     | 17           |       |
```

Comment:
we should also update the min version here? I think we dropped support to jdk8 with this change.

Reply:
this column is for xenial but we have deprecated it recently...